nextflow.config

```diff
@@ -144,7 +144,10 @@ try {
 }
 
 profiles {
-  conda { process.conda = "$baseDir/environment.yml" }
+  conda {
+    process.conda = "$baseDir/environment.yml"
+    conda.createTimeout = '1 h'
```

Comment:
Just out of interest. Why is this required?
I never had the creation of a Conda env fail.

Reply:
If the file system is share file system it will fail. At least in our nfs file system for the number of dependencies we have it fails.